carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h

```diff
@@ -213,13 +213,6 @@ class WMBroadcaster
    */
   ros::V_string invertParticipants(const ros::V_string& input_participants) const;
 
```

Comment:
need to declare ros::V_string combineParticipantsToVehicle(const ros::V_string& input_participants) const;
undo the change